test/models/getPenaltyDetails/lateSubmission/LSPDetailsSpec.scala

```diff
@@ -222,6 +256,12 @@ class LSPDetailsSpec extends SpecBase {
     result.get shouldBe modelWithBlankAppealLevel
   }
 
+  "be readable from JSON when the penaltyCategory is ' '" in {
+    val result = Json.fromJson(jsonRespresentingModelWithBlankCategory)(LSPDetails.reads)
+    result.isSuccess shouldBe true
+    result.get shouldBe model
+  }
+
   "be writable to JSON" in {
```

Comment:
Could there be a writable test?

Reply:
I'm not sure how I'd write a writable test, since there isn't an enum associated with it, so there wouldn't be a distinct model to write to json.

Reply:
Ah I was wrong... 
Maybe it's worth having some sort of IT test to cover receiving it as blank and then parsing it to 'point'